core/src/main/scala-2.13+/cats/instances/ScalaVersionSpecificParallelInstances.scala

```diff
@@ -0,0 +1,124 @@
+package cats
+package instances
+
+import cats.data._
+import cats.kernel.Semigroup
+import cats.syntax.either._
+import cats.{~>, Applicative, Apply, FlatMap, Functor, Monad, NonEmptyParallel, Parallel}
+import kernel.compat.scalaVersionSpecific._
```

Comment:
In all version specific code (i.e. not being cross compiled), you shouldn't need to import a `compat.scalaVersionSpecific._` right? which also means that you shouldn't need the `@suppressUnusedImportWarningForScalaVersionSpecific` below as well.

Reply:
I think you're right. Let me try removing all of these in the version-specific source trees.

Reply:
In some cases we might still want to use some of this stuff to minimize the diff between version-specific files? Like `IterableOnce`? I'm not sure—let me try it.

Reply:
Thanks for catching that, @kailuowang—there were only a couple of place we were using `IterableOnce` or `lazyZip` in version-specific code. Just pushed a commit removing  `scalaVersionSpecific` in all version-specific code (except where it's defined).